components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -87,6 +87,10 @@
 #include "url/url_canon_stdstring.h"
 #include "url/url_util.h"
 
+#include "brave/components/services/bat_ledger/bat_ledger_service_impl.h"
```

Comment:
Hm, this thing should be changed: `brave/components/services/bat_ledger/` should be treated as a private part of the service and shouldn't be included here.

Reply:
Fixed